boards/arm/stm32l562e_dk/stm32l562e_dk.dts

```diff
@@ -38,3 +38,9 @@
 &lptim1 {
 	status = "okay";
 };
+
+&spi3 {
+	pinctrl-0 = <&spi3_sck_pg9 &spi3_mosi_pb5 &spi3_miso_pb4>;
+	cs-gpios = <&gpioe 0 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+	status = "okay";
+};
```

Comment:
Could you move this to stm32l562e_dk_common.dtsi so that is is available on both secure and non secure ?